routing_common/pedestrian_model.cpp

```diff
@@ -52,553 +51,553 @@ double constexpr kSpeedPierKMpH = 4.0;
-  { {"highway", "living_street"},  kSpeedLivingStreetKMpH },
-  { {"highway", "steps"},          kSpeedStepsKMpH }, // *
-  { {"highway", "pedestrian"},     kSpeedPedestrianKMpH },
-  { {"highway", "footway"},        kSpeedFootwayKMpH },
-  { {"highway", "platform"},       kSpeedPlatformKMpH }, // *
+  { {"highway", "trunk"},          kSpeedTrunkKMpH,         true },
+  { {"highway", "trunk_link"},     kSpeedTrunkLinkKMpH,     true },
+  { {"highway", "primary"},        kSpeedPrimaryKMpH,       true },
+  { {"highway", "primary_link"},   kSpeedPrimaryLinkKMpH,   true },
+  { {"highway", "secondary"},      kSpeedSecondaryKMpH,     true },
+  { {"highway", "secondary_link"}, kSpeedSecondaryLinkKMpH, true },
+  { {"highway", "tertiary"},       kSpeedTertiaryKMpH,      true },
+  { {"highway", "tertiary_link"},  kSpeedTertiaryLinkKMpH,  true },
+  { {"highway", "service"},        kSpeedServiceKMpH,       true },
+  { {"highway", "unclassified"},   kSpeedUnclassifiedKMpH,  true },
+  { {"highway", "road"},           kSpeedRoadKMpH,          true },
+  { {"highway", "track"},          kSpeedTrackKMpH,         true },
+  { {"highway", "path"},           kSpeedPathKMpH,          true },
+  { {"highway", "bridleway"},      kSpeedBridlewayKMpH,     true },
+  { {"highway", "cycleway"},       kSpeedCyclewayKMpH,      true },
+  { {"highway", "residential"},    kSpeedResidentialKMpH,   true },
+  { {"highway", "living_street"},  kSpeedLivingStreetKMpH,  true },
+  { {"highway", "steps"},          kSpeedStepsKMpH,         true },
+  { {"highway", "pedestrian"},     kSpeedPedestrianKMpH,    true },
+  { {"highway", "footway"},        kSpeedFootwayKMpH,       true },
+  { {"highway", "platform"},       kSpeedPlatformKMpH,      true },
 };
 
 // Austria
 routing::VehicleModel::InitListT const g_pedestrianLimitsAustria =
 {
-  { {"highway", "primary"},        kSpeedPrimaryKMpH },
-  { {"highway", "primary_link"},   kSpeedPrimaryLinkKMpH },
-  { {"highway", "secondary"},      kSpeedSecondaryKMpH },
-  { {"highway", "secondary_link"}, kSpeedSecondaryLinkKMpH },
-  { {"highway", "tertiary"},       kSpeedTertiaryKMpH },
-  { {"highway", "tertiary_link"},  kSpeedTertiaryLinkKMpH },
-  { {"highway", "service"},        kSpeedServiceKMpH },
-  { {"highway", "unclassified"},   kSpeedUnclassifiedKMpH },
-  { {"highway", "road"},           kSpeedRoadKMpH },
-  { {"highway", "track"},          kSpeedTrackKMpH },
-  { {"highway", "path"},           kSpeedPathKMpH },
-  { {"highway", "cycleway"},       kSpeedCyclewayKMpH },
-  { {"highway", "residential"},    kSpeedResidentialKMpH },
-  { {"highway", "living_street"},  kSpeedLivingStreetKMpH },
-  { {"highway", "steps"},          kSpeedStepsKMpH }, // *
-  { {"highway", "pedestrian"},     kSpeedPedestrianKMpH },
-  { {"highway", "footway"},        kSpeedFootwayKMpH },
-  { {"highway", "platform"},       kSpeedPlatformKMpH }, // *
+  { {"highway", "primary"},        kSpeedPrimaryKMpH,       true },
```

Comment:
Am I right that g_pedestrianLimitsAustria is the same as g_pedestrianLimitsAustralia?